src/oss/paths/legacy_authorizations.yml

```diff
@@ -58,7 +58,7 @@ post:
     content:
       application/json:
         schema:
-          $ref: "../../common/schemas/AuthorizationPostRequest.yml"
+          $ref: "../../oss/schemas/LegacyAuthorizationPostRequest.yml"
```

Comment:
`LegacyAuthorizationPostRequest` needs added to the `schemas:` map at the bottom